### PR TITLE
Create models for the search job feature

### LINKED ISSUE #31: Create job search models
Create the classes: 
- Company
-  JobPost
- UserApplication

### PR DESCRIPTION
**Pull request by shakedch**
- add models for the search job
- define searchJob as an application 

Fixes: #31 

@bardielle @benipeled @guy9050

### CODE REVIEW

File: searchJob/admin.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+from django.apps import apps
+
+# Register your models here.# Register your models here.
```
Comment (bardielle) [inline searchJob/admin.py]: remove the comment

File: searchJob/admin.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+models = apps.get_models()
+for model in models:
```
Comment (bardielle) [inline searchJob/admin.py]: please import the explicit models and for each one register
like that you will know for any model that you're using and not register for all

File: searchJob/models.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+
+# Create your models here.
+
+
+class Company(models.Model):
+    company_name = models.CharField(max_length=100, primary_key=True)
```
Comment (bardielle) [inline searchJob/models.py]: what about two companies with the same name? 
you need another field that will be the primary key

File: searchJob/models.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    profile_description = models.TextField()
+    establishment_date = models.DateField()
+    contact_number = models.CharField(max_length=12, blank=True, null=True)
+    company_type = models.TextChoices('company_type', 'FINTECH BIOTECH MEDTECH FOODTECH AUTOMATIVE SW HW')
```
Comment (bardielle) [inline searchJob/models.py]: Remove it (FINTECH BIOTECH MEDTECH FOODTECH AUTOMATIVE SW HW)  to a consts that will be outside from the model

File: searchJob/models.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    compamy_url = models.URLField(max_length=200)
+
+    def __str__(self):
+        return self.company_name
+
+
+class JobPost(models.Model):
+    JobType_id = models.ForeignKey("user_notification.JobType", on_delete=models.CASCADE, blank=True, null=True)
+    company_id = models.ForeignKey(Company, on_delete=models.CASCADE)
```
Comment (bardielle) [inline searchJob/models.py:44]: you dont have company_id in the company model....

**Comment by bardielle** [comment]
@ifireball can you take a look?

**Comment by ifireball** [comment]
The code generally looks good to me, was this tested?

**Pull request merged**